pkg/systemd/init.js

```diff
@@ -613,7 +613,7 @@ $(function() {
 
             var since = "";
             if (timestamp)
-                since = cockpit.format(_("Since $0"), new Date(timestamp/1000).toLocaleString());
+                since = cockpit.format(_("Since $0"), moment(timestamp/1000).format('LLL'));
```

Comment:
For German I seem to get:
"Seit 30. Mai 2018 22:05"
The dot after 30 seems to be a bit misplaced, I don't think German usually does that. But we'll need a German to verify.

Reply:
@martinpitt said it's expected in German, so ignore.